### PR TITLE
add: test-cases

### PR DESCRIPTION
**Pull request by kavyakunder**
Test case for:
App.js
TodoList.js
Screenshot 2023-02-16 at 9 14 36 PM

Screenshot 2023-02-17 at 10 28 23 AM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { useEffect, useState } from "react";
 import { TextField, Button, Box } from "@mui/material";
 import List from "@mui/material/List";
-import TodoList from "./components/TodoList";
 import Typography from "@mui/material/Typography";
 import { useAppStyles } from "./App.style";
+import { TodoList } from "./components/TodoList";
+
```
Comment (shah-harshit) [inline src/App.js]: We don't need this empty space.

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +1,69 @@
...
+    const inputText = screen.getByTestId("input-text");
+    const btnAdd = screen.getByTestId("btn-add");
+
+    expect(headingElement).toBeInTheDocument();
+    expect(inputText).toBeInTheDocument();
+    expect(btnAdd).toBeInTheDocument();
+    expect(btnAdd).toBeDisabled();
+  });
+
+  it("render add functionality", async () => {
```
Comment (shah-harshit) [inline src/App.test.js]: Can we change the title, this doesn't seem relevant?

File: src/components/TodoList.jsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ function TodoList({ item, deleteFn, id, handleChange }) {
             alignItems="center"
           >
             <Grid item xs={8}>
-              <ListItem data-testid="list-item" key={id} paddingLeft={10}>
+              <ListItem data-testid="list-item" key={id} paddingleft={10}>
```
Comment (shah-harshit) [inline src/components/TodoList.jsx]: It should be `paddingLeft`

File: src/components/TodoList.test.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { TodoList } from "./TodoList";
+
+describe("Todo List", () => {
+  it("save an edit", () => {
```
Comment (shah-harshit) [inline src/components/TodoList.test.js]: Can we update this title?

File: src/components/TodoList.test.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+  it("render delete functionality", () => {
+    render(<TodoList {...mockProps} />);
+    const btnDelete = screen.getByTestId("btn-delete");
+
+    fireEvent.click(btnDelete);
+    expect(mockDelete).toHaveBeenCalled();
+  });
+
+  it("cancel an edit", () => {
```
Comment (shah-harshit) [inline src/components/TodoList.test.js]: Can we update this title?

File: src/components/TodoList.jsx (hunk 3 above)
Comment (kavyakunder) [reply src/components/TodoList.jsx]: for paddingLeft it gives warning in the console that "React does not recognize the `paddingLeft` prop on a DOM element" and suggests to use in lowercase

Comment (shah-harshit) [reply src/components/TodoList.jsx]: Which means `ListItem` may not accept paddingLeft. You need to apply it to the `Grid`.

**Review by shah-harshit — APPROVED** [review]

**Pull request merged**